.github/workflows/test.yml

```diff
@@ -8,7 +8,7 @@ jobs:
   linux:
     strategy:
         matrix:
-            nixpkgs: [ nixpkgs, nixpkgs-21.05, nixpkgs-21.11 ]
+            nixpkgs: [ nixpkgs ]
```

Comment:
Multi-nixpkgs tests have become problematic because the older nixpkgs don't contain `rustPlatform.bindgenHook` and provide older OpenSSL - meaning that we can either use multi-nixpkgs only for fast-tests (like in here) _or_ have conditional tests (i.e. don't run the `oqs` test on non-recent nixpkgs).

Since reducing the matrix is the simpler approach (and we still make sure Naersk remains backwards-compatible through fast-tests), I think it's the better approach.